public/apps/configuration/panels/get-started.tsx

```diff
@@ -0,0 +1,171 @@
+          to define how to retrieve and verify the user credentials, and how to fetch additional
+          roles from backend system if needed.
+        </EuiText>
+
+        <EuiSpacer />
+
+        <EuiFlexGroup gutterSize="s" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiButton fill iconType="popout" iconSide="right">
+              Create config.yml
+            </EuiButton>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              onClick={() => {
+                window.location.href = buildHashUrl(ResourceType.auth);
+              }}
+            >
+              Review authentication and authorization
+            </EuiButton>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </Fragment>
+    ),
+  },
+  {
+    title: 'Find a role that satisfies your security needs',
+    children: (
+      <Fragment>
+        <EuiText size="s" color="subdued" grow={false}>
+          A role defines cluster permissions, index permissions, and read/write access to a tenant.
+          <br />
+          You can make use of a build-in role, or create a custom role based on your needs.{' '}
+          <EuiLink external={true} href="/">
+            Learn More
+          </EuiLink>
+        </EuiText>
+
+        <EuiSpacer />
+
+        <EuiFlexGroup gutterSize="s" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              fill
+              onClick={() => {
+                window.location.href = buildHashUrl(ResourceType.roles);
+              }}
+            >
+              Explore existing roles
+            </EuiButton>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              onClick={() => {
+                window.location.href = buildHashUrl(ResourceType.roles, Action.create);
+              }}
+            >
+              Create new role
+            </EuiButton>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </Fragment>
+    ),
+  },
+  {
+    title: 'Map internal users or external identities to your role',
+    children: (
+      <Fragment>
+        <EuiText size="s" color="subdued" grow={false}>
+          Map users to a role to uptake its role settings. You can build an internal user database
+          within this plugin, or use a external entities to directly map to a role through an
+          external authentication system.{' '}
+          <EuiLink external={true} href="/">
+            Learn More
+          </EuiLink>
+        </EuiText>
+
+        <EuiSpacer />
+
+        <EuiFlexGroup gutterSize="s" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiButton fill>Map users to a role</EuiButton>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              onClick={() => {
+                window.location.href = buildHashUrl(ResourceType.users, Action.create);
```

Comment:
Can you confirm this is the expected URL?

Reply:
Yes. It is correct one. It is the same as the link of "Create internal user" button on the Internal users page.